fairroot/online/steer/FairRunOnline.cxx

```diff
@@ -217,12 +217,18 @@ Int_t FairRunOnline::EventLoop()
     auto const tmpId = GetEvtHeaderRunId();
 
     if (tmpId != fRunId) {
-        LOG(info) << "FairRunOnline::EventLoop() Call Reinit due to changed RunID (from " << fRunId << " to " << tmpId;
         fRunId = tmpId;
-        Reinit(fRunId);
-        if (!GetSource()->ReInitUnpackers()) {
-            LOG(fatal) << "FairRunOnline->EventLoop() ReInitUnpackers() failed!";
-            return 1;
+        if (!fStatic) {
+            LOG(info) << "FairRunOnline::EventLoop() Call Reinit due to changed RunID (from " << fRunId << " to "
```

Comment:
The `fRunId = tmpId;` in line 221/220 will make both the same.
So the LOG will only show the new RunID.

Maybe move the LOG above the assignment and make a second LOG in both if branches?